### DISCUSSION TITLE
Indent Animation: Start from cursor line?

### DISCUSSION THREAD

**Discussion by Kayzels** [folke/snacks.nvim#263] [Ideas]
Switching over from mini.indentscope, this was a cool feature it had. So, if I'm closer to the bottom of the scope, it will animate from my cursor going up and down until it finishes animating the scope. At the moment, the animation in snacks.indent seems to only work from the top of the scope down, regardless of the cursor position.

**Comment by folke** [comment]
Just added animation styles. `opts.indent.animate.style = "out"`. `"out"` is the default.

  * out: animate outwards from the cursor
  * up: animate upwards from the cursor
  * down: animate downwards from the cursor
  * up_down: animate up or down based on the cursor position
